VCFv4.4.tex

```diff
@@ -556,10 +556,22 @@ \section{INFO keys used for structural variants}
 \begin{samepage}
 The following INFO keys are reserved for encoding structural variants.
 In general, when these keys are used by imprecise variants, the values should be best estimates.
-When a key reflects a property of a single alt allele (e.g.\ SVLEN), then when there are multiple alt alleles there will be multiple values for the key corresponding to each allele (e.g.\ SVLEN=-100,-110 for a deletion with two distinct alt alleles).
+When present, per allele keys must be specified for all alt alleles (including non-structural alleles).
+The missing value should be used as a placeholder for the alt alleles for which the key does not have a meaningful value.
+The empty string is used to encode missing values in lists of strings.
+
```

Comment:
Since the files were renamed since this was submitted, could you please rebase against `VCFv4.4.draft.tex`?